google-http-client/src/test/java/com/google/api/client/util/SecurityUtilsTest.java

```diff
@@ -160,4 +166,51 @@ public void testVerifyX509() throws Exception {
   public void testVerifyX509WrongCa() throws Exception {
     assertNull(verifyX509(TestCertificates.BOGUS_CA_CERT));
   }
+
+  public void testCreateMtlsKeyStoreNoCert() throws Exception {
+    final InputStream certMissing =
+        getClass()
+            .getClassLoader()
+            .getResourceAsStream("com/google/api/client/util/privateKey.pem");
+    IllegalArgumentException exception =
+        assertThrows(
+            IllegalArgumentException.class,
+            new ThrowingRunnable() {
+              @Override
+              public void run() throws Throwable {
+                SecurityUtils.createMtlsKeyStore(certMissing);
+              }
+            });
```

Comment:
Use a try/catch block for asserting exceptions thrown:

```java
boolean thrown = false;
try {
  SecurityUtils.createMtlsKeyStore(certMissing);
  fail("should have thrown");
} catch (IllegalArgumentException e) {
  assertTrue(e.getMessage().contains("some message"));
  thrown = true;
}
assertTrue(thrown, "should have caught an IllegalArgumentException");
```

Reply:
done